docs/CONTRIBUTING.md

```diff
@@ -7,6 +7,11 @@ git clone git@github.com:kinvolk/lokomotive.git
 cd lokomotive
 ```
 
+## Requirements
+
+- Minimum Go version **1.4**
```

Comment:
Do you mean 1.14?

Reply:
Oh yes. Will update it in next commit as Suraj said